pkg/process/metadata/parser/service_test.go

```diff
@@ -117,6 +117,13 @@ func TestExtractServiceMetadata(t *testing.T) {
 			},
 			expectedServiceTag: "process_context:cat",
 		},
+		{
+			name: "kubelet heath check",
+			cmdline: []string{
+				"kubelet", "--address=1.1.1.1", "--cloud-provider=aws",
+			},
+			expectedServiceTag: "service:kubelet",
```

Comment:
We shouldn't let the agent define a `service` tag automatically, and instead continue using `process_context` as the key (which I believe is already the case for this CLI).

We initially had used `service` instead of `process_context` in this class but it adds complications in the backend / product when differentiating between customer-defined versus auto-detected services.

Reply:
https://github.com/DataDog/datadog-agent/pull/14771

Reply:
ah thanks for flagging this! I did not know the context behind this tag